yb-voyager/src/tgtdb/yugabytedb.go

```diff
@@ -1067,7 +1071,7 @@ func checkSessionVariableSupport(tconf *TargetConf, sqlStmt string) bool {
 				if !utils.AskPrompt("Are you sure you want to proceed?") {
 					utils.ErrExit("Aborting import.")
 				}
-				return true
+				return false // support is not there even if the target user doesn't have privileges to set this parameter.
```

Comment:
is this a bug that you caught?

Reply:
Not a bug, but a miss to mark the support of the parameter as false in case the permission is not available. But this doesn't affect the setting of session parameter because we already have a handling at conn pool level to ignore the Permissioned denied.